README.md

```diff
@@ -1,3 +1,7 @@
+This is a fork of [tmuxinator/tmuxinator](https://github.com/tmuxinator/tmuxinator). In this fork I am trying to add some new features. If a feature is ready, I will send it back as a pull request.
```

Comment:
My fault. This should only be part of the main branch, but never be merged to the feature branch. I wanted to have a disclaimer on the main branch of my fork to make clear it is only a fork and I did not invent tmuxinator.